scripts/table_updates/update_data_table.py

```diff
@@ -24,307 +24,438 @@
 
 from utilities import *
 
-TABLE_INFO = {"primary": ('syn23285911',"table_type='data'"),
-              "irr": ('syn21446696',"table_type='data' and double_curated is true")}
+TABLE_INFO = {
+    "primary": ("syn23285911", "table_type='data'"),
+    "irr": ("syn21446696", "table_type='data' and double_curated is true"),
+}
+
+
+def get_main_genie_clinical_sample_file(
```

Comment:
Nit: this is great for this function, but I think we have this particular "get main genie file from release" function used across all GENIE operations.  In the future, when we re-design this whole thing, it might be worthwhile to add all GENIE helper functions in one package.